graph/pull.go

```diff
@@ -553,6 +554,8 @@ func (s *TagStore) pullV2Tag(eng *engine.Engine, r *registry.Session, out io.Wri
 
 	}
 
+	out.Write(sf.FormatStatus(repoInfo.CanonicalName+":"+tag, "The image you are pulling has been verified - This is a tech preview, don't rely on it for security yet."))
```

Comment:
Ah ok that @fredlf I will ping you in a followup PR sorry we didn't wait.

On Wed, Jan 21, 2015 at 5:59 PM, Fred Lifton <notifications@github.com>
wrote:

> In graph/pull.go
> <https://github.com/docker/docker/pull/10235#discussion_r23350953>:
>
> > @@ -553,6 +554,8 @@ func (s *TagStore) pullV2Tag(eng *engine.Engine, r *registry.Session, out io.Wri
> >
> >  	}
> >
> > +	out.Write(sf.FormatStatus(repoInfo.CanonicalName+":"+tag, "The image you are pulling has been verified - This is a tech preview, don't rely on it for security yet."))
>
> Arnaud asked me to look at this and I'm just getting to it now.
> I would suggest the following: "The image you are pulling has been
> verified. *Important:* image verification is a tech preview feature and
> should not be relied on to provide security."
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/10235/files#r23350953>.
>